src/plugins/expressions/public/types/functions.ts

```diff
@@ -47,3 +49,11 @@ export interface FunctionHandlers {
 }
 
 export type AnyExpressionFunction = ExpressionFunction<string, any, any, any>;
+
+export interface FunctionExamples {
```

Comment:
Maybe we could add the same comment here too so it shows up if this interface is ever imported directly?